source/docs/v3/integrations/sail.blade.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Laravel Sail integration
+extends: _layouts.documentation
+section: content
+---
+
+# Laravel Sail {#sail}
+
+## Central domains
+
+If you're using Laravel Sail, no changes are needed, default values are good to go:
+
+```php
+// config/tenancy.php
+
+'central_domains' => [
+    '127.0.0.1',
+    'localhost',
+],
+```
```

Comment:
What's this section for? It made sense in the quickstart, but if this is an integration guide, it shouldn't mention things that **don't** need to be done.

Reply:
FIxed